pkgs/by-name/um/umu-launcher-unwrapped/no-umu-version-json.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/Makefile.in b/Makefile.in
+index b82053d..37db8c9 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -50,7 +50,7 @@ fix_shebangs:
+ umu/umu_version.json: umu/umu_version.json.in
+ 	$(info :: Updating $(@) )
+ 	cp $(<) $(<).tmp
+-	sed 's|##UMU_VERSION##|$(shell git describe --always --long --tags)|g' -i $(<).tmp
++	sed 's|##UMU_VERSION##|@version@|g' -i $(<).tmp
```

Comment:
Could we upstream a change where `UMU_VERSION` can be specified as a make flag (defaulting to using `git describe`) so that we can just specify that flag instead of patching the makefile?

Of course, this is fine for now.

Reply:
If they accept the PR, then yes, but I won't be the one to do it.